src/main/kotlin/com/exactpro/th2/http/client/Main.kt

```diff
@@ -227,10 +355,16 @@ data class Settings(
     val sessionAlias: String,
     val auth: IAuthSettings? = null,
     val validateCertificates: Boolean = true,
+    val useTransport: Boolean = false,
+    val batchByGroup: Boolean = true,
```

Comment:
Do we need it at all? I think we must batch by group - there is no option anymore because cradle might not store messages if we group by alias only